alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +26,16 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec:
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	go get -u github.com/securego/gosec/cmd/gosec
```

Comment:
we cannot depend on the master branch. This must be a specific version. Otherwise, CI will break randomly because gosec changed.

Reply:
Done